twindle-cli/renderer/pdf/index.js

```diff
@@ -8,7 +8,8 @@ async function generatePDF(tweets, outputPath) {
   const htmlContent = await renderTemplate({ thread: tweets.data, common: tweets.common }, "Thread");
 
   // creates the pdf from html and saves it to Twindle.pdf
-  await createPdf(outputPath, htmlContent);
+  if(tweets.data.length > 0)
+    await createPdf(outputPath, htmlContent);
```

Comment:
@Mira-Alf long-term, I would think that we can avoid the renderer knowing that it is rendering tweets, my thought is that we can do this check in the main index.js when we get the tweet results from api.

Reply:
Sounds good. I can clean it up the next time I commit something.